Bottom: 60cd8d2d0d9e2308225776d66a87d3f29a40a266
Top:    eedc0e20c7b18772b3a791c6ab676afd106f8c62
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:43:17 +0000

find.c: Alter LookupConnection to lookup inc./excl. rat-connections in one pass


---

diff --git a/src/find.c b/src/find.c
index d80d249..3c1cca2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3087,8 +3087,7 @@ ListStart (int type, void *ptr1, void *ptr2, void *ptr3, int flag)
  * also the action is marked as undoable if AndDraw is true
  */
 void
-LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
-                  bool AndRats)
+LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int connected_flag, int found_flag)
 {
   void *ptr1, *ptr2, *ptr3;
   char *name;
